commitizen.config.js

```diff
@@ -0,0 +1,38 @@
+const types = [
+  {
+    value: 'feat',
+    name: 'feat:     A new project test or feature is being added.'
+  },
+  {
+    value: 'fix',
+    name: `fix:      Modifying an existing projects tests, fixing a bug
+            or text.`
+  },
+  {
+    value: 'docs',
+    name: 'docs:     Documentation only changes to readme, guides, etc.'
+  },
+  {
+    value: 'test',
+    name: `test:     Adding missing tests to tooling (NOT to be used for
+            projects tests).`
+  },
+  {
+    value: 'chore',
+    name: `chore:    Changes to the build process or auxiliary tools
+            and libraries such as documentation generation.`
+  },
+  {
+    value: 'revert',
+    name: 'revert:   Revert a commit.'
+  }
+];
+
+const scopes = ['tests', 'tools', 'scripts'];
+
+module.exports = {
+  types,
+  scopes,
+  allowCustomScopes: true,
+  allowBreakingChanges: ['feat', 'fix', 'perf', 'refactor']
```

Comment:
Is refactor a valid type? I don't see it in the list above?

Reply:
@tchaffee it's how it is in curriculum repo

Delete it or add it to types?

Reply:
That helps me understand where this came from. Let's leave everything as close to the curriculum repo as possible and only move away from that if it starts to cause us problems. The eventual goal is to share one standard.